forge-gui/res/cardsfolder/d/disa_the_restless.txt

```diff
@@ -5,7 +5,7 @@ PT:5/6
 T:Mode$ ChangesZone | ValidCard$ Permanent.Lhurgoyf+YouOwn | Origin$ Library,Hand,Exile,Command,Stack | Destination$ Graveyard | TriggerZones$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ Whenever a Lhurgoyf permanent card is put into your graveyard from anywhere other than the battlefield, put it onto the battlefield.
 SVar:TrigChangeZone:DB$ ChangeZone | Defined$ TriggeredCard | Origin$ Graveyard | Destination$ Battlefield
 T:Mode$ DamageDoneOnce | ValidSource$ Creature.YouCtrl | TriggerZones$ Battlefield | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigToken | TriggerDescription$ Whenever one or more creatures you control deal combat damage to a player, create a Tarmogoyf token. (It's a {1}{G} Lhurgoyf creature with "Tarmogoyf's power is equal to the number of card types among cards in all graveyards and its toughness is equal to that number plus 1.")
-SVar:TrigToken:DB$ CopyPermanent | DefinedName$ Tarmogoyf
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ tarmogoyf
```

Comment:
There was a reason for this so that we don't create TokenScript for each of them
The Effect itself should take care of that getting the right token image

Reply:
It didn't work for me. It always gave me the key is malformed.

Reply:
Yeah it should have been fixed on the source (in the Copy Permanent Effect)